packages/popover/src/vaadin-popover-overlay-mixin.js

```diff
@@ -59,6 +59,8 @@ export const PopoverOverlayMixin = (superClass) =>
         return;
       }
 
+      this.removeAttribute('arrow-centered');
```

Comment:
**question:** I'm not sure why it's only needed for horizontal positions. I tested it and it magically works well, but I would still appreciate an explanation.

Reply:
This is a result of different logic for `start` and `end` vertical positions where we [always center the overlay](https://github.com/vaadin/web-components/blob/01919cfff3e58a39671f2911092f29fb014be6da/packages/popover/src/vaadin-popover-overlay-mixin.js#L89-L90). This aligns with how `vaadin-tooltip` positions work, although I don't remember why it was done this way.